modules/apigee/variables.tf

```diff
@@ -50,6 +50,7 @@ variable "environments" {
     description     = optional(string, "Terraform-managed")
     deployment_type = optional(string)
     api_proxy_type  = optional(string)
+    type = optional(string)
```

Comment:
Can you sort members alphabetically? We usually do that, but for some reason, it slip under the radar here.

Reply:
Sure!  Thanks, first time contributing to this repo so bear with me. :-)